karma.conf.js

```diff
@@ -30,7 +43,27 @@ module.exports = function (karmaConfig) {
 
     // Test results reporter to use
     // Possible values: 'dots', 'progress', 'junit', 'growl', 'coverage'
-    reporters: ['progress'],
+    reporters: karmaReporters,
```

Comment:
I added the variable so I can test for `testConfig.coverage === true` which is set by the grunt coverage task [here](https://github.com/meanjs/mean/pull/985/files#diff-595bf3fa2348192244b0319be33066b8R227)